notesapi/v1/views.py

```diff
@@ -141,15 +141,30 @@ class AnnotationDetailView(APIView):
 
     UPDATE_FILTER_FIELDS = ('updated', 'created', 'user', 'consumer')
 
+    def _fetch(self, annotation_id, reply_id=None):
```

Comment:
I think it's better to just raise the `models.DoesNotExist`. It's more idiomatic, and also avoids the ugly pattern of a function that returns a variety of object types (sometimes a `Note`, other times `None`, for example).

This makes the caller easier to understand as well, because it catches the exception and returns the appropriate response code. The reader doesn't need to refer to `_fetch` to figure out why `None` might come back.